core/src/main/scala/org/bitcoins/core/wallet/fee/FeeUnit.scala

```diff
@@ -168,6 +168,8 @@ case class SatoshisPerVirtualByte(currencyUnit: CurrencyUnit)
     SatoshisPerVirtualByte
 
   override def toString: String = s"$toLong sats/vbyte"
+
+  lazy val toSatoshisPerKW: SatoshisPerKW = SatoshisPerKW(currencyUnit * 4000)
```

Comment:
We should add a sanity unit test for this -- basically a couple of simple examples.